apps/frontend/utils/tableFormatting.tsx

```diff
@@ -85,7 +89,7 @@ const getLocationColumnSetup = (
         <FormattedMessage id={`forms_table.headers.${params.field}`} />
       </Typography>
     ),
-    valueFormatter: value =>
+    valueGetter: value =>
```

Comment:
Can you explicit why this change is needed?

Reply:
The return value of `valueGetter` is used for sorting and it's being passed in the `valueFormatter` or `renderCell`. With this change:
1. We can use `params.value` in the `renderCell` without the need to duplicate the logic to construct the value.
2. The sorting is based on the displayed value (the actual name of the location) and not on the code ("01-002") and behaves as expected (unless we need a custom order based on the code and not alphanumerical order).

From [docs](https://mui.com/x/react-data-grid/column-definition/#value-formatter):
> Note, that the value returned by valueFormatter is only used for rendering purposes. Filtering and sorting are based on the raw value (row[field]) or the value returned by [valueGetter](https://mui.com/x/react-data-grid/column-definition/#value-getter).

Reply:
We can use the same approach here to calculate `formattedList` once in the `valueGetter`

https://github.com/WFP-VAM/dmp/blob/74ec186e365f7ffe50ac225422c544ea5df6296b/apps/frontend/utils/tableFormatting.tsx#L121-L139

I skipped this as I thought it was out of scope and required more testing, but we can make that change since we are working on this area

Reply:
Ok, worth exploring. Two things come to mind though:
1. Ideally, for sorting, we might actually want to order by number of villages instead of the alphabetical order of the first village (which doesn't really mean anything)
2. We still need to display the tooltip